src/Compilers/Core/Portable/Generated/FlowAnalysis.Generated.cs

```diff
@@ -5,6 +5,7 @@
 using System;
 using System.Collections.Immutable;
 using Microsoft.CodeAnalysis.Operations;
+using Microsoft.CodeAnalysis.PooledObjects;
```

Comment:
could be reverted?

Reply:
No, it could not. It will be needed when we eventually port over flow analysis nodes with children.